mibs/oneaccess/ONEACCESS-SYS-MIB.mib

```diff
@@ -0,0 +1,483 @@
+--          *****************************

```

Comment:
Hi
Could you rename the mib file without the ".mib" extension ?
Thx

Reply:
Hi ! 
it's good now